exporters/otlp/common/src/main/java/io/opentelemetry/exporter/internal/TlsUtil.java

```diff
@@ -72,5 +127,29 @@ public static X509TrustManager trustManager(byte[] trustedCertificatesPem) throw
     }
   }
 
-  private TlsUtil() {}
+
+  /**
+   * Reads pem file with private key and certification chain for the key.
+   * @param filePath path to pem file containing private key with chain
+   * @return Array of bytes where first item (indexed 0) is private key all others are for chain
+   * @throws IOException when there is problem reading provided file
+   */
+  public static byte[] loadPemFile(String filePath) throws IOException {
+    BufferedReader bufferedReader = Files.newBufferedReader(new File(filePath).toPath(), UTF_8);
+    String line = bufferedReader.readLine();
+    while (line != null && !line.startsWith("-----BEGIN ")) {
+      line = bufferedReader.readLine();
+    }
+    line = bufferedReader.readLine();
+    if (line != null) {
+      StringBuilder buf = new StringBuilder();
+      while (!line.startsWith("-----END ")) {
```

Comment:
What happens if the file is malformed and ends before we find `-----END`? Looks like a NPE would get thrown once we reach the end of the file.

Reply:
added check also for this case